migration-client.js

```diff
@@ -104,11 +104,29 @@ class MigrationClient {
     await this.#applyEvents(docEvents, collEvents);
   }
 
+  /**
+   * List all collections in the source database.
+   *
+   * @returns {Promise<string[]>}
+   */
+  async listSourceCollections() {
+    return await this.#sourceClient.query(
+      Select(
+        "data",
+        Map(Paginate(Collections(), { size: 100000 }), (ref) =>
```

Comment:
Maybe this should be a much smaller page size and the help text can say what it is.  Otherwise the tool will run 4eva.

`[optional] the comma-delimited list of Collection names to be sync'ed (max: 100)`

Reply:
And if there are more than 100 (i.e. if this comes back with an `after` token), maybe we fail?

Reply:
If I can add the `--migrate-all` option we can stop to ask the customer if they want to proceed given that many Collections.

I'd also add a `-y/--yes` as well to bypass the question for testing or for folks who are brave.

How about that?